### PR TITLE
New package: UZ7HO.SoundModem version 1.14

### LINKED ISSUE #115467: [Package Request]: SoundModem
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: http://uz7.ho.ua/packetradio.htm
Publisher: UZ7HO
Package Name: SoundModem
Description: The software Packet-Radio TNC.
Package Version: 1.14
Installer URL: http://uz7.ho.ua/modem_beta/soundmodem114.zip

### PR DESCRIPTION
**Pull request by Dragon1573**
## Description

- Resolve https://github.com/microsoft/winget-pkgs/issues/115467

## Checks

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/123420)

### CODE REVIEW

File: manifests/u/UZ7HO/SoundModem/1.14/UZ7HO.SoundModem.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+PackageIdentifier: UZ7HO.SoundModem
+PackageVersion: "1.14"
+InstallerType: zip
+Installers:
+- Architecture: x86
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: soundmodem.exe
+    PortableCommandAlias: SoundModem
+  InstallerUrl: http://uz7.ho.ua/modem_beta/soundmodem114.zip
```
Comment (Dragon1573) [inline manifests/u/UZ7HO/SoundModem/1.14/UZ7HO.SoundModem.installer.yaml:13]:
```suggestion
  InstallerUrl: https://uz7.ho.ua/modem_beta/soundmodem114.zip
```

File: manifests/u/UZ7HO/SoundModem/1.14/UZ7HO.SoundModem.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: UZ7HO.SoundModem
+PackageVersion: "1.14"
+PackageLocale: en-US
+Publisher: UZ7HO
+PublisherUrl: http://uz7.ho.ua/
```
Comment (Dragon1573) [inline manifests/u/UZ7HO/SoundModem/1.14/UZ7HO.SoundModem.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://uz7.ho.ua/
```

**Review by Dragon1573 — COMMENTED** [review]
Try switch to HTTPS protocol.

File: manifests/u/UZ7HO/SoundModem/1.14/UZ7HO.SoundModem.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: UZ7HO.SoundModem
+PackageVersion: "1.14"
+PackageLocale: en-US
+Publisher: UZ7HO
+PublisherUrl: http://uz7.ho.ua/
+Author: UZ7HO
+PackageName: SoundModem
+PackageUrl: http://uz7.ho.ua/packetradio.htm
```
Comment (Dragon1573) [inline manifests/u/UZ7HO/SoundModem/1.14/UZ7HO.SoundModem.locale.en-US.yaml:11]:
```suggestion
PackageUrl: https://uz7.ho.ua/packetradio.htm
```

**Comment by Dragon1573** [comment]
- https://github.com/microsoft/winget-pkgs/pull/123419#issuecomment-1774115636

URL without SSL/TLS is expected.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**